src/CoreWCF.Templates/src/templates/CoreWCFService/IService.cs

```diff
@@ -0,0 +1,61 @@
+﻿#if(!impliciteUsings)
+using CoreWCF;
+using System;
+using System.Runtime.Serialization;
+#endif
+
+namespace CoreWCFService
```

Comment:
Its not usual for templates to have multiple classes etc in one file. But in this case, I think its appropriate. It makes it very clear that this is all the contract stuff, and so you don't have to go fishing to find it.